src/diffpy/srmise/baselines/nanospherical.py

```diff
@@ -247,8 +247,8 @@ def getmodule(self):
     for tup in zip(r, val, *outjac):
         for t in tup:
             if t is None:
-                formatted_t = "%s" % None
+                formatted_t = f"{None}"
                 print(formatted_t.ljust(10))
```

Comment:
reduce to one line, just put the string in the print statement to make it more readable.